frameworks/sveltekit.md

```diff
@@ -13,7 +19,56 @@ For `Type declarations`, `Prompt for update` and `Periodic SW Updates` go to [Sv
 :::
 
 ::: tip
-You should remove all references to [SvelteKit service worker module](https://kit.svelte.dev/docs/modules#$service-worker) to disable it on your application.
+If you're using `0.1.*` version of `SvelteKitPWA`, you should remove all references to [SvelteKit service worker module](https://kit.svelte.dev/docs/service-workers) to disable it on your application.
+:::
+
+## Generate Custom Service Worker
+
+From version `0.2.0`, `SvelteKitPWA` plugin will delegate your custom service worker build to `SvelteKit`, and so, you can use `src/service-worker.js`: in that case, you don't need to configure the `filename` in pwa plugin options.
+
+You will need to exclude the service worker registration from the `SvelteKit` configuration if you're using any virtual module:
```

Comment:
is it virtual modules from SvelteKit or virtual modules from vite PWA plugin or either?